test/studies/shootout/submitted/regexdna-submitted.graph

```diff
@@ -1,5 +1,5 @@
-perfkeys: real
-graphkeys: release
-files: regexdna-release.dat
+perfkeys: real real real
```

Comment:
Should be comma separated

Reply:
That's so weird that I missed that... While typing it, I was thinking "it's so dumb that these aren't comma-separated.  I'm not sure where I got it into my head that they weren't.  Not from any of the other files in this directory, apparently.  But also weird: in testing, I didn't notice anything barking at me that it was wrong (though maybe it just didn't bark loudly enough for me to notice... I was cooking dinner at the time).